<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OGRaySpY</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_a_spectrum"/>
    <addaction name="actionGenerate_report"/>
    <addaction name="actionP_lots"/>
    <addaction name="actionPandas_dataframe"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionLanguages"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Settings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open_a_spectrum">
   <property name="text">
    <string>&amp;Open a spectrum</string>
   </property>
  </action>
  <action name="actionGenerate_report">
   <property name="text">
    <string>Generate &amp;report</string>
   </property>
  </action>
  <action name="actionP_lots">
   <property name="text">
    <string>P&amp;lots</string>
   </property>
  </action>
  <action name="actionPandas_dataframe">
   <property name="text">
    <string>Pandas &amp;dataframe</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionLanguages">
   <property name="text">
    <string>Languages</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
